# date:2007-04-03 owner:elaine type:bug bugster:6524819 state:upstream
# 15378753 SUNBT6524819 Gaim will crash when select protocol for new account
#
# This change is already present upstream.

--- pidgin-2.13.0/pidgin/gtkaccount.c
+++ pidgin-2.13.0/pidgin/gtkaccount.c
@@ -474,6 +474,7 @@ add_login_options(AccountPrefsDialog *di
 		g_object_ref(G_OBJECT(dialog->protocol_menu));
 #else
 		gtk_widget_ref(dialog->protocol_menu);
+                gtk_widget_show(dialog->protocol_menu);
 #endif
 	}
 
